.github/workflows/build-wheels-linux-arm64-self-hosted.yml

```diff
@@ -1,8 +1,8 @@
 name: linux-arm64-dispatch
 
 on:
-  schedule:
-     - cron: '0 0 * * 0,3'
+  # schedule:
```

Comment:
@jakub-kocka Is there a specific reason for keeping the commented code? I suggest removing it if it's not necessary.

Reply:
You are right, it could be removed. However there will be another following PR (final for the refactor) which will clear the whole repository of unused code. (I have decided to do it this way to see if everything works when switched to the refactored version)

So these files where the comments are now will be deleted anyway so it is not necessary to do it in this PR.